test/e2e/mock/dl.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable camelcase */
```

Comment:
```suggestion
```

Reply:
if we don't change the function names I can't remove